### PR TITLE
[ticket/15274] Document parameters for custom functions

### PR DESCRIPTION
**Pull request by javiexin**
Also, show example of compatibility with multi step process

### CODE REVIEW

File: development/migrations/data_changes.rst [hunk 1]
```diff
@@ -179,6 +179,77 @@ Example
         return $value + $limit;
     }
 
+Passing parameters to custom functions
+--------------------------------------
+
+.. note::
+
+    This functionalilty is only available from 3.2.2 onwards
```
Comment (callumacrae) [inline development/migrations/data_changes.rst]: functionalilty -> functionality

File: development/migrations/data_changes.rst [hunk 2]
```diff
@@ -179,6 +179,77 @@ Example
...
+            return;
+        }
+
+        // There are still more topics to query, return the next start value
+        return $value + $limit;
+    }
+
+.. note::
+
+    The function called must accept the specified parameters, plus the $value param as the last one
```
Comment (callumacrae) [inline development/migrations/data_changes.rst]: This sentence doesn't really make sense but I'm not sure what it's supposed to say so can't suggest anything better :(

File: development/migrations/data_changes.rst (hunk 1 above)
Comment (javiexin) [inline development/migrations/data_changes.rst]: Oops, sorry.

File: development/migrations/data_changes.rst (hunk 2 above)
Comment (javiexin) [inline development/migrations/data_changes.rst]: Ok, I tried to clarify, let's see if this time is more understandable.

File: development/migrations/data_changes.rst [hunk 3]
```diff
@@ -248,7 +248,8 @@ Fully compatible with multi step process
 
 .. note::
 
-    The function called must accept the specified parameters, plus the $value param as the last one
+    To support the multi-step process, the function called must accept the explicit parameters, 
+	plus an additional last parameter that will receive the receive the return of the previous step.
```
Comment (Nicofuma) [inline development/migrations/data_changes.rst]: ` receive the receive the return` you have 2 extra words

Comment (javiexin) [inline development/migrations/data_changes.rst]: I don't know what happens to me today...  Apologies again!

**Comment by javiexin** [comment]
Given that the main ticket has been merged into Master, and not 3.2, then the documentation is wrong.
It states this is available since 3.2.2, and it won't.

**Pull request merged**